yugabyte/src/yugabyte/ysql/append_rr.clj

```diff
@@ -0,0 +1,38 @@
+(ns yugabyte.ysql.append-rr
+  "Implementation for REPEATABLE READ transaction isolation"
+  (:require [clojure.java.jdbc :as j]
+            [clojure.tools.logging :refer [info]]
+            [yugabyte.ysql.append :as append]
+            [yugabyte.ysql.client :as c]))
+
+(defrecord InternalClient []
```

Comment:
I don't think we need 2 (or 3 even?) almost identical files/records, just make it configurable.

Reply:
Fixed